test/specs/templates.e2e.ts

```diff
@@ -634,14 +639,18 @@ describe('Templates', async () => {
   step('Create Lightning Web Component Test', async () => {
     // Delete previous test file
     const workbench = await (await browser.getWorkbench()).wait();
-    const sidebar = workbench.getSideBar();
-    const treeViewSection = await sidebar.getContent().getSection(projectName);
-    await treeViewSection.expand();
-    const lwcTestFolder = await treeViewSection.findItem('__tests__');
-    await lwcTestFolder?.select();
-    let testItem = await treeViewSection.findItem('lightningWebComponent1.test.js');
-    await testItem?.select;
-    await browser.keys([CMD_KEY, 'Delete']);
+    const pathToLwcTest = path.join(
```

Comment:
@CristiCanizales why the change to use shell command instead of UI?

Reply:
UI wasn't working, and the deletion itself is not what we want to test but I need to delete the file so the command can successfully run, as when there's already a test file the command fails.